### ISSUE TITLE
Create validation and test sets for Do Not Sell link and US Privacy String detection

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-optmeowt#219]
I will create validation sets for tuning the Do Not Sell link and US Privacy String detection. I will also create test sets for measuring the analysis accuracy of the links and Strings, e.g., using precision, recall, and F-1, once we are done with the tuning. Each set will have 50 to 100 items. I have to see how difficult it is to find sites with such links and Strings manually. I will probably start with the Tranco list going top to bottom.

**Comment by SebastianZimmeck** [comment]
OK, I have created a test set for Do Not Sell link and US Privacy String detection. I extended @OliverWang13's [Google Sheet](https://docs.google.com/spreadsheets/d/19Wi2PaPsEOfiSdeaOEdSNg5QCzJbRmc7f5rHhXYzO6E/edit?usp=sharing).

I am leaving it to you, @OliverWang13 (and possibly @stanleymarkman) to create a validation set for US Privacy Strings as you see fit.

As background info: in supervised machine learning, we can use three data sets:
- training set: for training the algorithm/techniques on the data
- validation set: new data not in the training set to fine-tune the algorithm/techniques
- test set: new data not in the training or validation sets to evaluate the performance of the algorithm/techniques

Our techniques are rule-based and do not use machine learning. So, we do not need to train and, thus, do not need a training set.

Please only run our code on the test set when you are fairly confident that the Do Not Sell link and US Privacy String detection works well. Otherwise, we will need to create a new test set and keep iterating.

Also, the validation set for the US Privacy String cannot contain any of the Privacy String sites already in the test set. Otherwise, we would tune towards our test set. So, when you validate, make sure you are not using any data that is also in the test set.
